site/docs-md/guides/deep-links.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Deep Links
+description: Implement deep linking functionality in an iOS and Android app
+url: /docs/guides/deep-links
+contributors:
+  - dotnetkow
+---
+
+# Deep Linking with Universal and App Links
+
+**Platforms**: Web, iOS, Android
+
+Universal links (iOS) and App Links (Android) offer the ability to take users directly to specific content within a native app (commonly known as deep linking).
+
+When users tap or click on a deep link, the mobile operating system sends the user directly into your app without routing through the device's web browser or website first. If the app isn't installed, then the user is directed to the website. If the user navigates directly to the website, they remain on the website.  This makes deep links an excellent feature for cross-platform apps built for the web, iOS, and Android.
```

Comment:
Again, deep links don't work on web, they are regular links. I don't think we can say it's cross-platform if Capacitor's Web App plugin doesn't respond to `appUrlOpen`